playbooks/roles/ocp-upgrade/defaults/main.yaml

```diff
@@ -0,0 +1,7 @@
+---
+# defaults file
+
+upgrade_image: ""
+pause_time: 90
+delay_time: 600
```

Comment:
Better to create upgrade_vars.yaml in defaults and including it here

Reply:
@bpradipt So there will be `upgrade_vars.yaml `and it will be included in `main.yaml` right user defaults dir?
Also, you want to move all the vars or only time vars to upgrade_vars.yaml?

Reply:
basically I would like to see vars required for upgrade in its own file and the same being included in the playbook.

Reply:
@bpradipt Above `defaults/main.yaml` is specific to upgrade role if that is your concern. Also under examples, we have a separate file for upgrade_vars. Similarly, install and upgrade playbooks are kept independent as well.

Reply:
@Prajyot-Parab thanks for the clarification.